### ISSUE TITLE
formatNumber not working with parenthesis in the format

### ISSUE THREAD

**Issue by taki9** [amcharts/amcharts4#1672]
Hi!

I try to use the formatNumber function the following way:
`` `{value.formatNumber("${format}")}` `` where format is `const format = '(#.)'`
It is not working when the format contains parenthesis.

I think the problem is caused by the regex in the getTagValue function of the Sprite class.

**Comment by martynasma** [comment]
Indeed. Parenthesis are currently not supported in in-line formatting functions.

I'm marking this as a bug so that we can get back to it.

At this point I'm not entirely sure when this will be implemented, sorry.
